packages/algob/src/lib/tx.ts

```diff
@@ -141,6 +142,42 @@ async function sendAndWait (
   return await deployer.waitForConfirmation(txInfo.txId);
 }
 
+/**
+ * Make transaction parameters and update ASA and SSC params
+ * @param deployer Deployer object
+ * @param t Execution parameters
+ * @param index index of current execParam
+ * @param txIDMap Map for index to name
+ */
+async function mkTx (
+  deployer: Deployer,
+  t: rtypes.ExecParams,
```

Comment:
`t` looks too cryptic in a separate function. Can we rename this to `tx`/`txn`/`currentTx`/`singleTx` ?

Reply:
yes